index.js

```diff
@@ -18,9 +23,33 @@ module.exports = {
     }
   },
 
+  treeForVendor(defaultTree) {
+    var trees = [];
+
+		if (defaultTree) {
+			trees.push(defaultTree);
+    }
+    
+    let trackJsPath = path.join(this.project.root, 'node_modules', 'trackjs');
+
+    let browserVendorLib = fbTransform(new Funnel(trackJsPath, {
+      files: ['tracker.js'],
+      destDir: 'trackjs',
+    }));
+  
+    trees.push(browserVendorLib);
+    
+    return mergeTrees(trees);
+  },
+
   included(app) {
-    this._super.included(app);
+    this._super.included.apply(this, arguments);
+
+    // allow addon to be nested - see: https://github.com/ember-cli/ember-cli/issues/3718
+    while (app.app) {
+      app = app.app;
+    }
 
-    app.import(require.resolve('trackjs'));
+    app.import(this.treePaths.vendor + '/trackjs/tracker.js');
```

Comment:
Why not use string interpolation instead?

Reply:
solved